ReactiveCocoa/Swift/Signal.swift

```diff
@@ -953,6 +972,18 @@ public func zip<A, B, C, D, E, F, G, H, I, J, Error>(a: Signal<A, Error>, b: Sig
 		|> map(repack)
 }
 
+/// Zips the values of all the given signals, in the manner described by
+/// `zipWith`.
```

Comment:
Can you please document that this operator will never send events if the sequence is empty?